openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -7219,11 +7220,21 @@ def get_graph_def_from_file(graph_filepath):
             import traceback
             traceback.print_exc()
 
+    def get_TFLiteConverter():
+        if dont_save_keras_model:
+            run_model = tf.function(lambda x : model(x))
```

Comment:
Limit the variable scope of `model` and `dont_save_keras_model` to within a function. This is a bit verbose, but it should be passed as a function argument.